src/script.ts

```diff
@@ -100,7 +100,7 @@ export function loadScript(options: ILoadScriptOptions = {}): Promise<HTMLScript
       const src = script.getAttribute('src');
       if (src !== url) {
         // potentially trying to load a different version of the API
-        reject(new Error(`The ArcGIS API for JavaScript is already loaded (${src}).`));
+        reject(new Error(`The ArcGIS API is already loaded (${src}).`));
```

Comment:
```suggestion
        reject(new Error(`The ArcGIS SDK is already loaded (${src}).`));
```

I think any change to this error message is _technically_ a breaking change, but it should not affect most developers. We could leave this message as is until the next breaking change, but I don't anticipate any of those now that we are in maintenance mode. I'm good to modify it w/o a major version bump, but we should make an explicit note about it in the changelog.

Reply:
@tomwayson I reverted this change. It didn't feel right making a breaking change and not incrementing the version. I'll open an issue to update the error messages as part of the 4.26 release.